modules/redis/src/main/scala/zio/redis/options/SortedSets.scala

```diff
@@ -82,6 +82,8 @@ trait SortedSets {
 
   type MemberScores[+M] = Chunk[MemberScore[M]]
 
+  sealed case class RankScore(score: Double, rank: Long)
```

Comment:
```suggestion
  sealed case class RankScore(rank: Long, score: Double)
```

We should consider introducing newtypes (maybe through zio-prelude) to represent things like `Rank` consistently and "cost-free". That's a topic for a separate PR, though.